.github/workflows/awscleanup.yaml

```diff
@@ -0,0 +1,44 @@
+name: Daily AWS Cleanup Bot
+
+on:
+  schedule:
+    - cron: '0 8 * * *' # Runs daily at 8AM UTC
```

Comment:
@cdesiniotis @tariq1890 
suggestion :- 
 **- cron: "0 */4 * * *"**    

Since we are deleting only e2e job resources (ci-*), shouldn’t we run this job every 4 hours? 
with this change, Our cleanup bot will destroy only those resources that are completed. 
Instead of running it daily at 8 AM UTC, we should schedule it to run every 4 hours.